### PR TITLE
Enhancements to the Linux workshop

### PR DESCRIPTION
**Pull request by Verisimilitude11**
Corrected grammatical errors, added more content, and altered some minor formatting inconsistencies.

### CODE REVIEW

File: content/english/linux-basics/_index.md [hunk 1]
```diff
@@ -20,19 +20,19 @@ This tutorial requires:
 
 {{% /notice %}}
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/zfdlPZYlgtk" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zfdlPZYlgtk" alt="A YouTube video that introduces the Linux Workshop" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (Verisimilitude11) [inline content/english/linux-basics/_index.md:23]: Added alt text

File: content/english/linux-basics/_index.md [hunk 2]
```diff
@@ -20,19 +20,19 @@ This tutorial requires:
...
 
-## Welcome
+## Welcome!
```
Comment (Verisimilitude11) [inline content/english/linux-basics/_index.md:25]: Added exclamation point

File: content/english/linux-basics/activity-10.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ difficulty: "Intermediate"
 weight: 13
 ---
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/LPcQW4oGK6g" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/LPcQW4oGK6g" alt="A YouTube video on Linux Basics Activity 2" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (ozhang22) [inline content/english/linux-basics/activity-10.md:9]:
```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/LPcQW4oGK6g" alt="A YouTube video on Linux Basics Activity 10" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: content/english/linux-basics/activity-6.md [hunk 4]
```diff
@@ -35,7 +35,7 @@ grep -R "Do. Or do not. There is no try."
 
 ### Why does the secret-message file look like that?
 
-*Remember that file we saw with the strange name? It was called `malware.sh`. We should go find it again and take a closer look at it. I think this was left by the hackers!*
+*Remember that file we saw with the strange name? It was called `malware.sh`. We should go find it again and take a closer look at it. I think that the hackers  left this!*
```
Comment (ozhang22) [inline content/english/linux-basics/activity-6.md:38]:
```suggestion
*Remember that file we saw with the strange name? It was called `malware.sh`. We should go find it again and take a closer look at it. I think that the hackers left this!*
```

**Review by ozhang22 — APPROVED** [review]
Looks good!

**Pull request merged**